db/migrate/20221017163325_reset_theses_cache_counters.rb

```diff
@@ -0,0 +1,10 @@
+class ResetThesesCacheCounters < ActiveRecord::Migration[6.1]
+  def up
+    Thesis.all.each do |thesis|
+      Thesis.reset_counters(thesis.id, :authors)
```

Comment:
I'm semi-nervous to allow this to happen during the pre-deploy step. It's not wrong by any means and will do what we need, but I sort of prefer to let the app start normally and then console in to do this one-time thing with the slight downside that the reports would have wrong data for the few minutes that it takes us to console in and run the script. This isn't a blocking comment, just expressing my "hmmm" thought on the two approaches.

Reply:
As discussed in our team meeting, I'm planning to replace this with a rake task. In the unlikely event that we have to reset counters more than once, having a documented rake task will help us remember how (and why) to do it.

Reply:
The solution you put in works great. Thanks!

Reply:
Ha, I meant that for the administrate bit... but the rake task is great too :)